.github/workflows/ohos.yml

```
@@ -321,10 +321,10 @@ jobs:
           git fetch upstream main
           echo "RUN_BENCHER_OPTIONS=--branch try \
           --hash $(git rev-parse HEAD~1) \
-          --branch-start-point main \
-          --branch-start-point-hash $(git merge-base upstream/main HEAD) \
-          --start-point-clone-thresholds \
-          --branch-reset" >> "$GITHUB_ENV"
+          --start-point main \
+          --start-point-hash $(git merge-base upstream/main HEAD) \
+          --point-clone-thresholds \
```

```suggestion
          --start-point-clone-thresholds \
```

This change was probably a mistake, right?

---

Yes, a mistake

---

I think the fix got lost during the rebase. Failed in the try run here.